<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Siddharth Jain</title>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-YVFTDPHXJC"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-YVFTDPHXJC');
</script>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">General</div>
<div class="menu-item"><a href="index.html" class="current">Home</a></div>
<!--<div class="menu-item"><a href="About.html">About Me</a></div>-->
<div class="menu-item"><a href="CV_Siddharth.pdf">Curriculum Vitae</a></div>
<div class="menu-item"><a href="publications_updated.html">Publications</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Siddharth Jain</h1>
</div>
<table class="imgtable"><tr><td>
<img src="kk.jpg" alt="That's me" width="180px" height="180px" />&nbsp;</td>
<td align="left"><p><a href="https://paradise.caltech.edu/~sidjain/">Siddharth Jain</a><br />
Research Scientist<br />
PhD Advisor: <a href="http://paradise.caltech.edu/bruck.html">Professor Jehoshua (Shuki) Bruck </a><br />
<a href="http://ee2.caltech.edu">Electrical Engineering Department</a><br />
<a href="http://www.caltech.edu">Caltech</a><br />
Email: sidjain at caltech dot edu <br />
Phone: +1-626-652-1958<br />
Photo Credits: Prachi Parihar<br />
<!--ErdÅ‘s Number: 2<br />-->
</p>
</td></tr></table>
<!--<b>I am on the 2020-2021 academic job market. Please feel free to contact me! Here is my <a href = "CV_Siddharth_Caltech_2.pdf">CV</a>, <a href = "Teaching_Statement_final_3.pdf">Teaching Statement</a> and <a href = "Research_Statement_final_main.pdf">Research Statement</a>.</b>--> 
<!--<b>The most exciting phrase in science is not 'Eureka!' But 'That's funny ...'</b> - <i>Isaac Asimov </i>-->
<!--<h2>Research Interests</h2>
I am interested in understanding the interplay between <b>computation, information</b> and <b>evolution</b>. This quest has led me to focus on theoretical problems related to finding the fundamental limits of evolution using <b>information theory</b> and <b>theory of computation</b> tools to applied problems in <b>healthcare</b> primarily focusing on <b>cancer classification</b> and <b>viral evolution</b>. Having used data science/ML methods on datasets and realizing their noisy and biased nature, I am also interested in designing algorithms and approaches with theoretical guarantees which ensure <i>robustness</i> in <b>data science/ML</b>. My other research interests include <i>connections</i> between <b>data compression</b> & <b>learning</b> and <b>DNA Storage</b>.  related to ensuring The human genome is a complex set of information with many evolution secrets hidden within. Unfolding these secrets has applications in predicting disease, discerning ancestry, materializing DNA storage, and designing synthetic biology devices for computation. In the past, many studies have only focused on small portions of DNA for evolutionary inferences, as the whole human genome was not readily available. Following the success of the human genome project and super exponential progress in sequencing technology, the whole human genome of high quality can now be availed at a very low cost. Genomic studies have largely focused on making evolution inference out of the genome by comparing observed phenotypes (like diseases, physical traits, race, gender,etc.), a scientific philosophy motivated by backward inference. However, now that we have the whole genome available, forward inference about evolution can be made directly. 

My interests lie in using the ideas and philosophies of information theory, coding theory, theoretical computer science, and signal processing to make this forward inference by understanding the evolution channel of each genome. More precisely, I want to describe an individual's genome with a persona by viewing it as a signal of mutation activity. 
<p> </p>-->
<!--<h2>Research Interests</h2>
<ul><li>Quality, Privacy and Value of Data<br></ul></li>
<ul><li>Constrained Systems - Coding, Networks<br></ul></li>
<ul><li>Disease Genomics, DNA Storage and Computational Biology<br></ul></li>-->
<h2>News!</h2>
<p><b>Apr 30, 2021: </b><i>Our paper titled ``Synthesizing new expertize via collaboration,'' got accepted in <b>IEEE ISIT 2021</b><a href = "https://arxiv.org/abs/2107.07054"> [arXiv Link]</a></i>
<p><b>Sep 25, 2020: </b><i>Our paper titled ``Robust Correction of Sampling Bias Using Cumulative Distribution Functions,'' got accepted in <b>NeurIPS 2020</b></i><a href = "https://arxiv.org/abs/2010.12687"> [arXiv Link]</a>
<p><b>July 27, 2020:</b><i> New paper on bioRxiv, "Predicting the Emergence of SARS-CoV-2 Clades</i> <a href = "https://www.biorxiv.org/content/10.1101/2020.07.26.222117v1"> [Link]</a>
<p><b>May 7, 2020:</b> <i> New paper on arXiv, "Coding for Optimized Writing Rate in DNA Storage"</i> <a href = "https://arxiv.org/abs/2005.03248"> [Link]</a><a href = "ISIT_final_Sidd.mp4"> [Video Link]</a>
<p><b>Apr 22, 2020:</b> <i> New paper on arXiv, "CodNN - Robust Neural Netowrks from Coded Classification"</i> <a href = "https://arxiv.org/abs/2004.10700"> [Link]</a>
<p><b>Mar 31, 2020:</b> <i> Three new papers accepted in <b>IEEE ISIT 2020</b>.</i>
<ul><li><i>"Coding for Optimized Writing in DNA Storage.''</i></ul></li>
<ul><li><i>"CodNN - Robust Neural Networks from Coded Classification.''</i></ul></li>
<ul><li><i>"What is the Value of Data? on Mathematical Methods for Data Quality Estimation.''</i></ul></li>
<p><b>Jan 17, 2020:</b> <i> Paper accepted in <b>NVMW 2020</b>, "Coded Deep Neural Networks for Robust Neural Computation"</i>
<p><b>Jan 9, 2020:</b> <i> New paper on arXiv, "What is the value of data? On Mathematical Methods for Data Quality Estimation"</i><a href = "https://arxiv.org/abs/2001.03464"> [Link]</a>
<p><b>Sept 24, 2019:</b> <i>Our work related to Cancer Classification was accepted to <a href = "https://lmrl-bio.github.io/"><b>LMRL</b> </a>workshop at <b>NeurIPS 2019</b></i>
<p><b>June 20, 2019:</b> <i>We filed a patent on "Mutation Profile and related labeled Genomic Components, Methods and Systems".</i>
<p><b>May 17, 2019:</b> <i>Our work "Data Equals Money, But How Much? On Mathematical Methods for Data Pricing" got accepted in <b>CodML</b> workshop in <b>ICML</b></i>
<p><b>Apr 1, 2019:</b> <i> I successfully defended my PhD thesis "Decoding the Past"</i><a href = "https://thesis.library.caltech.edu/11436/"> [Link]</a>
<p><b>Mar 21, 2019:</b> <i>I gave an invited talk at Conference on Information Sciences and Systems (CISS) at John Hopkins</i>
<p><b>Feb 13, 2019:</b> <i>I gave an invited graduation day talk at ITA Workshop in San Diego</i>
<p><b>Jan 11, 2019:</b> <i>Our article on Cancer Classification using healthy DNA is available on bioRxiv</i> <a href = https://doi.org/10.1101/517839>[Link]</a></p>
<p><b>Jan 11, 2019:</b> <i>Our article on the statistical bias present in short tandem repeats in amplified samples on TCGA is now available on bioRxiv</i> <a href = https://doi.org/10.1101/518878>[Link]</a></p>  
<h2>Services</h2>
<ul>
<li>Reviewer for IEEE Transactions on Information Theory</li></ul>
<ul><li>Reviewer for IEEE Transactions on Communications</li></ul>
<ul><li>Reviewer for ACM Transactions on Algorithms</li></ul>
<ul><li>Reviewer for AAAS Science Advances</li></ul>
<ul><li>Reviewer for Frontiers in Physiology</li></ul>
<ul><li>Reviewer for IEEE Communication Letters</li></ul>
<ul><li>Reviewer for IEEE International Symposium on Information Theory (ISIT), NeurIPS, UAI.</li></ul>
<!--<h2>Current Research</h2>
<h3>Data Quality and Value</h3>
<p><a href = "https://www.linkedin.com/in/netanel-raviv-5a74a8108/"> Netanel Raviv </a>, <i>Siddharth Jain</i>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>
<b>Data Equals Money, But How Much? On Mathematical Methods of Data Pricing</b></br>
<i>CodML Workshop, ICML 2019</i>
<h3>Disease Risk Estimation</h3>
<p><i>Siddharth Jain</i>, B. Mazaheri,<a href = "https://www.linkedin.com/in/netanel-raviv-5a74a8108/"> N. Raviv</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>
<b>Mutation Profile and related labeled Genomic Components, Methods and Systems</b></br>
<i>Patent application filed, 2019.</i> 
<p><i>Siddharth Jain</i>, B. Mazaheri,<a href = "https://www.linkedin.com/in/netanel-raviv-5a74a8108/"> N. Raviv</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>
<b>Cancer Classification from Healthy DNA using Machine Learning</b></br>
<i>bioRxiv</i><a href = https://doi.org/10.1101/517839> [Link]</a>
<p><i>Siddharth Jain</i>, B. Mazaheri,<a href = "https://www.linkedin.com/in/netanel-raviv-5a74a8108/"> N. Raviv</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>
<b>Short Tandem Repeats Information in TCGA is Statistically Biased by Amplification</b></br>
<i> bioRxiv</i><a href = https://doi.org/10.1101/518878> [Link]</a>
<h3>Balanced Sequences</h3>
Many genomes including our own human genome share a property termed as <b>2nd Chargaff Rule</b>. According to this rule the number of occurences of a k-mer is almost equal to the number of occurences of it's reverse complement. For example, #A = #T, #C = #G, #AC = #GT and so on. This property is observed to hold upto a certain k known as K-Limit which depends on the length of genome. For human genome, the K-Limit is 10. In this project, we investigate models based on reversed tandem and interspered duplications for sequence generation that can give sequences satisfying the 2nd Chargaff Rule. We find fundamental limit onthe number of generations required to generate such <b> balanced </b> sequences for a given K-Limit. We compare the sequences generated by these models with real genome sequences and find results that give valubale insights into the evolution of genomes.
<ul>
<li>
<p><i>Siddharth Jain</i>,<a href = "https://www.linkedin.com/in/netanel-raviv-5a74a8108/"> N. Raviv</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>
<b>Attaining the 2nd Chargaff Rule by Tandem Duplications</b><a href = "Chargaff_2.pdf">[pdf]</a></br>
<i>accepted in 2018 IEEE International Symposium on Information Theory (ISIT)</i>
</ul>
</li>
<!--<h3>Early prediction of cancer </h3>
We develop algorithms using ideas from information and coding theory an statistics to estimate duplication and mutation history in tandem repeat regions of DNA. Given large amount of DNA data for different individuals, we preprocess each individual's DNA data by extracting mutation rate, history and copy number as the <b>features</b>. We use these features to differentiate individuals from different populations, carrying diseases (such as different cancers) using tools from Machine Learning. The databases used here are <a href="http://www.internationalgenome.org/home">1000GenomeBrowser</a>, <a href="https://cancergenome.nih.gov/">The Cancer Genome Atlas (TCGA)</a>.
<ul>
<li>
<p><i>Siddharth Jain</i>, <a href = "https://www.linkedin.com/in/netanel-raviv-5a74a8108/"> N. Raviv</a>, B. Mazaheri, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>
<b>On the probability of onset of cancer using mutation rate and copy number</b><br/>
<i>in preparation</i>
</ul>
</li>
<h3>String Duplication Systems</h3>     
Motivated by the phenomenon of tandem duplications observed in DNA, we investigate the diversity of sequences that can be generated by a series of tandem duplications applied on a seed. This diversity is characterized by mathematical measures of capacity and expressiveness. Given a sequence, we calculate upper and lower bounds on the number of steps required to reduce the sequence by omitting  tandem repeats to its seed. We also develop  algorithms that perform near optimal reduction.
<ul><li><p><i>Siddharth Jain</i>, <a href = "http://farnoud.info/"> F. Farnoud</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>
<b>Capacity and Expressiveness of Genomic Tandem Duplication</b><a href = "CapExp.pdf"> [pdf]</a><br/> <i>IEEE Transactions on Information Theory, vol 63, no. 10, pp. 6129-6138, October 2017.</i><br/><i>shorter version in Proceedings of 2015 IEEE International Symposium on Information Theory (ISIT), pp. 1946-1950, Hong Kong.</i></p>
</ul>
</li>
<ul>
<li>
<p><a href = "https://en.wikipedia.org/wiki/Noga_Alon">Noga Alon</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a>, <a href= "http://farnoud.info/">F. Farnoud</a>, <i> Siddharth Jain
</i> </br> <b>Duplication Distance to the Root for Binary Sequences</b><a href = "DupDist.pdf">[pdf]</a><br/><i>IEEE Transactions on Information Theory, vol 63, no. 12, pp. 7793-7803, December 2017.</i></p>
</ul>
</li>
<ul>
<li>
<p><a href = "https://en.wikipedia.org/wiki/Noga_Alon">Noga Alon</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a>, <a href= "http://farnoud.info/">F. Farnoud</a>, <i> Siddharth Jain</i></br> <b>On the Duplication Distance of Binary Strings</b><a href = "Duplication Distance 2016">[pdf]</a><br/><i> in Proceedings of 2016 IEEE International Symposium on Information Theory (ISIT), pp. 260-264 , Barcelona, Spain. </i></p>
</ul>
</li>
<h3>Codes for <i>live</i>-DNA Storage</h3> 
The advances in CRISPR have led to the realization of storing information inside the DNA of a living organism. As the cell replicates, this information can become erroneous due to mutation errors. Recovery of the stored information requires design of error correcting schemes that can correct these <i>evolutionary</i> errors. We investigated an evolution channel with unbounded duplication errors and came up with the error correcting codes. We also investigated errors caused by substitution mutations alongside duplications to come up with sphere packing bounds.
<ul>                                                                                                                                                                                                           
<li>                                                                                                                                                                                                           
<p><i>Siddharth Jain</i>, <a href = "http://farnoud.info/"> F. Farnoud</a>, <a href = "http://www.ee.bgu.ac.il/~schwartz/"> M. Schwartz</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>                                                                                                                                                                                                      
<b>Noise and Uncertainty in String-Duplication Systems</b><a href = "Noise_Unc.pdf">[pdf]</a><br/>                                                                                                             <i>in Proceedings of 2017 IEEE International Symposium on Information Theory (ISIT), pp. 3120-3124, Aachen, Germany.</i><br/></p>                                                                              
</ul>                                                                                                                                                                                                          
</li>                                                                                                                                                                                                          
<ul>                                                                                                                                                                                                         
<li>                                                                                                                                                                                                         
<p><i>Siddharth Jain</i>, <a href = "http://farnoud.info/"> F. Farnoud</a>, <a href = "http://www.ee.bgu.ac.il/~schwartz/"> M. Schwartz</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br>                                                                                                                                                                                                     
<b>                                                                                                                                                                                                          
Duplication-Correcting Codes for Data Storage in the DNA of a Living Organism</b><a href = "DupCorrect.pdf">[pdf]</a><br/>                                                                                   
<i>IEEE Transactions on Information Theory, vol 63, no. 8, pp. 4996-5010, August 2017. </i><br/><i>shorter version in Proceedings of 2016                                                                    
IEEE International Symposium on Information Theory (ISIT), pp. 1028-1032, Barcelona, Spain. </i></p>                                                                                                        
</ul>                                                                                                                                                                                                         
</li>
<h2>Past Research</h2>
<h3> Match Lengths, Zero Entropy and Large Deviations </h3>
We investigate the match length expression in the context of Sliding Window Lempel-Ziv Algorithm for zero entropy sequences. We also prove large deviation property for recurrence times and propose entropy estimator based on them under certain mixing conditions.
<ul>                                                                                                                                                                                                            
<li><p> <i>Siddharth Jain</i>,<a href= "http://www.iitk.ac.in/ee/faculty/rkbansal.shtml"> R. K. Bansal</a></br>                                                                                              
<b>On Match Lengths, Zero Entropy and Large Deviations - with Application to Sliding Window Lempel-Ziv Algorithm</b><a href = "http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6951479&queryText
%3DOn+Match+Lengths%2C+Zero+Entropy+and+Large+Deviations+-+with+Application+to+Sliding+Window+Lempel-Ziv+Algorithm">[Link]</a><a href = "Zero_Entropy_LDP.pdf">[pdf]</a></b><br/>                            
<i>IEEE Transactions on Information Theory, vol. 61, no. 1, pp. 120-132, January 2015.</i></p>                                                                                                               
</li>                                                                                                                                                                                                        
</ul>                                                                                                                                                                                                        
<ul>                                                                                                                                                                                                         
<li><p><i>Siddharth Jain</i>, <a href = "http://www.iitk.ac.in/ee/faculty/rkbansal.shtml">R. K. Bansal</a><br/>
<b>On Large Deviation Property of Recurrence Times</b><a href = "http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6620752&queryText%3DOn+Large+Deviation+property+of+Recurrence+Times">[Link]</a>
<a href = "LDP.pdf">[pdf]</a><br/> <i>Proceedings of IEEE Intenational Symposium on Information Theory (ISIT) 2013, pp. 2880-2884, Istanbul, Turkey.</i></p>
</li>
</ul>
<ul>
<li><p><i>Siddharth Jain</i>, <a href = "http://www.iitk.ac.in/ee/faculty/rkbansal.shtml">R. K. Bansal</a><br/>
<b>On Match Lengths and Asymptotic Behavior of Sliding Window Lempel-Ziv Algorithm for Zero Entropy Sequences</b><a href = "http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6620753&queryText%3
On+Match+Lengths+and+Asymptotic+Behavior+of+Sliding+Window+Lempel-Ziv+Algorithm+for+Zero+Entropy+Sequences">[Link]</a><a href = "Zero_Entropy.pdf">[pdf]</a><br/><i>Proceedings of IEEE International Syposium 
on Information Theory (ISIT) 2013, pp. 2885-2889, Istanbul, Turkey.</i></p>
</li>
</ul>

<h3>Fractional Calculus based approaches in Control</h3>
We propose a fractal optimal controller for heart rate control through pacemaker. The dynamics of the the heart rate are modeled by fractional differential equation. We also use a similar approach to perform power management for multi-voltage and frequency islands multiprocesser platforms.
<ul>
<li><p><a href = "http://users.ece.cmu.edu/~pbogdan/"> Paul Bogdan</a>, <i>Siddharth Jain</i>, Kartikeya Goyal, <a href = "http://www.ece.cmu.edu/directory/department/faculty/M/Radu_Marculescu_109.html">Radu Marculescu</a></br> 
<b>Implantable Pacemakers Control and Optimization via Fractional Calculus Approach: A Cyber Physical Systems (CPS) Perspective</b><a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6197385&contentType=Conference+Publications&searchField%3DSearch_All%26queryText%3DImplantable+Pacemakers+Control+and+Optimization+via+Fractional+Calculus+Approach%3A+A+Cyber+Physical+System+.LB.CPS.RB.+Perspective">[Link]</a> <a href="ICCPS.pdf">[pdf]</a><br />
<i>International Conference on Cyber Physical Systems (ICCPS), pp. 23-32, CPS Week 2012, Beijing, China. </i></p>
</li>
</ul>
<ul>
<li><p><a href = "http://users.ece.cmu.edu/~pbogdan/"> Paul Bogdan</a>, <i>Siddharth Jain</i>, <a href = "http://www.ece.cmu.edu/directory/department/faculty/M/Radu_Marculescu_109.html">Radu Marculescu</a></br> 
<b>Pacemaker Control of Heart Rate Variability: A CPS Perspective</b><a href = "http://dl.acm.org/citation.cfm?id=2435246">[Link]</a><a href = "Pacemaker_TECS.pdf">[pdf]</a></br>
<i>ACM Transactions on Embedded and Computing Systems (TECS), vol. 12, no. 1s, Article 50, March 2013.</i></p>
</li>
</ul>
<ul>
<li><p><a href= "http://users.ece.cmu.edu/~pbogdan/"> Paul Bogdan</a>, <a href = "http://www.ece.cmu.edu/directory/department/faculty/M/Radu_Marculescu_109.html">Radu Marculescu</a>, <i>Siddharth Jain</i> </br>
<b>Dynamic Power Management for Multi-domain Processor Systems-on-Chip Platforms: An Optimal Control Approach</b><a href = "http://dl.acm.org/citation.cfm?id=2504904">[Link]</a><a href = "VFI_TODAES.pdf">[pdf]</a><br/>
<i>ACM Transactions on Design Automation of Electronic Systems (TODAES), vol. 18, no. 4, Article 46, October 2013.</i></p>
</li>
</ul>
<ul>
<li><p><a href = "http://users.ece.cmu.edu/~pbogdan/"> Paul Bogdan</a>, <a href = "http://www.ece.cmu.edu/directory/department/faculty/M/Radu_Marculescu_109.html">Radu Marculescu</a>, <i>Siddharth Jain</i> and Rafael Tornero Gavila, </br> 
<b>An Optimal Control Approach to Power Management for Multi-Voltage and Frequency Islands Multi Processor platforms under highly variable workloads</b><a href ="http://www.ece.cmu.edu/news/story/2012/06/marculescu_receives_best.html"><b>[Best Paper Award]</b></a><a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6209260&contentType=Conference+Publications&searchField%3DSearch_All%26queryText%3DAn+Optimal+Control+Approach+to+Power+Management+for+Multi-Voltage+and+Frequency+Islands+Multi+Processor+platforms+under+highly+variable+workloads">[Link]</a> <a href="NOCS.pdf">[pdf]</a><br />
<i>International Syposium on Networks-On-Chip (NOCS) 2012, pp. 35-42, Lyngby, Denmark.</i></p>
</li>
</ul>
<h2> Talks </h2>
<ul>
<li> 
<b>Duplication Channel of The Genome.</b><br/><i>Conference on Information Sciences and Systems, John Hopkins, March 2019</i> [Invited]</p>
</ul>
<ul>
<li>
<b> Decoding the Past.</b><br/><i>Information Theory and Applications San Diego, Feb 13, 2019</i> [Invited]</p>
</ul>
<ul>
<li>
<b> Decoding the Past.</b><br/><i>Molecular Programming Project (MPP) Workshop, Boston, Massachusetts, December 9-11, 2016</i> [Invited]</p>
</ul>

<ul>
<li>
<b> Biological Information Channel.</b><br/><i>IPAM Computational Genomics Summer Institute (CGSI), UCLA, July 2016</i></p>
</ul>
</li>
<ul>                                                                                                                                                                                                         
<li>                                                                                                                                                                                                         
<b> Duplication Correcting Codes for DNA Storage.</b><br/><i>Molecular Programming Project (MPP) Workshop, Seattle, Washington, January 15-18, 2016</i> [Invited]</p>                                            
</ul>                                                                                                                                                                                                         
<h2>Posters</h2>
<ul>
<li>
<p><i>Siddharth Jain</i>, <a href = "http://farnoud.info/">F. Farnoud</a>, <a href = "http://www.ee.bgu.ac.il/~schwartz/">M. Schwartz</a>, <a href = "http://paradise.caltech.edu/bruck.html">Jehoshua Bruck</a></br> 
<b>Capacity and Diversity of Tandem Duplication.</b> <br/><i>Invited Poster in Molecular Programming Project (MPP) Workshop, San Francisco, California, January 9-11, 2015.</i></p>
</ul>
</li>
-->
<div id="footer">
<div id="footer-text">
Page generated by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
<!-- hitwebcounter Code START -->
<a href="http://www.hitwebcounter.com/" target="_blank">
<img src="http://hitwebcounter.com/counter/counter.php?page=4681666&style=0038&nbdigits=6&type=page&initCount=0" title="" Alt=""   border="0" >
</a>
</html>
